recipes/nglview/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: nglview
+  version: "0.5.dev"
+
+source:
+  git_url: https://github.com/arose/nglview
```

Comment:
Can you use a fixed version like this one `https://github.com/arose/nglview/archive/v0.4.tar.gz`